core/input/util.js

```diff
@@ -14,6 +14,20 @@ function isEdge() {
     return navigator && !!(/edge/i).exec(navigator.userAgent);
 }
 
+// Get the height of a un-styled line of text
+export function getScrollLineHeight() {
```

Comment:
Since this is rather heavy it should do the work when loaded and store the result in a variable, which can then efficiently be returned on each call.

Reply:
good idea!